dataverse-webapp/src/test/java/edu/harvard/iq/dataverse/search/SolrQueryCreatorTest.java

```diff
@@ -159,6 +171,16 @@ public void constructQuery_CombinedQuery() {
         return Lists.newArrayList(testValue1);
     }
 
+    private List<SearchField> createSelectOneSearchFields() {
+        SelectOneSearchField testValue1 = new SelectOneSearchField("selectOneValue1", "selectOneValue1", "desc");
+        testValue1.setCheckedFieldValue("selectOneValue1");
+
+        SelectOneSearchField testValue2 = new SelectOneSearchField("selectOneValue2", "selectOneValue2", "desc");
+        testValue2.setCheckedFieldValue("selectOneValue2");
```

Comment:
If all values wouldn't be the same then tests would check for more potential errors in the code.

Reply:
Maybe we can also differentiate between:
`new SelectOneSearchField("selectOneValue2", ...` and `.setCheckedFieldValue("selectOneValue2");`?

Reply:
We could but I don't think we should since one represents list of possible values to be checked, and the other that checked value. It wouldn't expect to be able to have checked value that is not on the list of options.

Reply:
From what I understand (feel free to correct me) in `new SelectOneSearchField("selectOneValue2", "selectOneDisplayValue2", "desc2");` you define values specific to field type (for example: `subject`, `Obszar tematyczy` and some description) and in `setCheckedFieldValue` value from controlled vocab (`for example `Chemistry`). So these can be different.

Reply:
And the list of correct values (that can be selected is in: `SelectOneSearchField.listLabelAndValue`)

Reply:
Yes, you're right. I've mistaken creating just `SelectOneSearchField` with propagating it with actual data.